Documentation/quickstart/running.rst

```diff
@@ -0,0 +1,101 @@
+.. include:: /substitutions.rst
+.. _running:
+
+Running
+=======
+
+Embedded boards have different ways to get your program onto them and get them running. This guide assumes your board
+has a JTAG connector, and you have a JTAG hardware debugger like a
```

Comment:
Wouldn't it be better to use a Nucleo board for this example which does not require an external programmer?
Maybe we can start with this easy case and then mention a board requiring a JTAG/SWD debugger on a different board.

Reply:
@v01d Let's add Nucleo or other boards in a different PR.

Reply:
Ok!